pkg/systemtemplate/import.go

```diff
@@ -25,8 +25,9 @@ import (
 )
 
 var (
-	templateFuncMap = sprig.TxtFuncMap()
-	t               = template.Must(template.New("import").Funcs(templateFuncMap).Parse(templateSource))
+	templateFuncMap      = sprig.TxtFuncMap()
+	t                    = template.Must(template.New("import").Funcs(templateFuncMap).Parse(templateSource))
+	preBootstrapTemplate = template.Must(template.New("import").Funcs(templateFuncMap).Parse(preBootstrapTemplateSource))
```

Comment:
How different does it look to use `if` expressions in the go template instead of having multiple sources? I'd prefer not to maintain multiple templates but understand if the alternative is more unwieldy.